cogs/kill.py

```diff
@@ -38,7 +38,7 @@ async def confirm_shutdown_button_callback(self, _: discord.Button, interaction:
     )
     async def cancel_shutdown_button_callback(self, _: discord.Button, interaction: discord.Interaction) -> None:  # noqa: E501
         """When the cancel button is pressed, delete the message."""
-        logger.debug("\"Cancel\" button pressed. %s", interaction)
+        logger.debug('"Cancel" button pressed. %s', interaction)
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes